.jenkins/jenkins.sh

```diff
@@ -143,7 +143,12 @@ export FV3_STENCIL_REBUILD_FLAG=False
 export TEST_DATA_HOST="${TEST_DATA_DIR}/${experiment}/"
 export EXPERIMENT=${experiment}
 if [ -z ${JENKINS_TAG} ]; then
-    export JENKINS_TAG=${JOB_NAME//[,=\/]/-}-${BUILD_NUMBER}
+    if [ ${#JOB_NAME} > 122 ]; then
```

Comment:
Just for my understanding, the `md5sum` guarantees a job name length of 32, and the check is for a length >= 123 because our prefixes are ~5 characters, e.g., `daint`, or `gce`?

Reply:
The max docker tag is 128 characters. I picked 122 to leave enough characters for a really large unrealistic build number. We could probably have this be 124 or 125 and be fine, or could hash the whole thing with the build number. But i thought it would be nice if at least some part of the tag had useful information. With the multiconfiguration projects, we are getting tags like v0.3.0-gnu7-mpich314-nocuda-developfv3core-regressions-cron-backend-numpy-experiment-c12_6ranks_standard-runaction-run_regression_tests-slave-gce-cpu-2
Oh actually, I should lower that threshold more, to ~75!

Reply:
Good idea!